Automattic/o2
# Dropdown Actions: Change the short link action to being copied to the clipboard

I kept clicking on this link, which takes me to the post and the window location is the full URL. I thought I'd see if I could change that behaviour so the short link is copied to the clipboard. I also added the shake animation to the icon.

I'm not sure whether the code is organised in the right way. I've added the CSS to the module, but the JS is in the main view for posts, which seems wrong. I'm happy to refactor as needed!

## Testing

- On a WordPress test site install the [P2-breathe theme](https://o2app.files.wordpress.com/2018/04/p2-breathe.zip), along with the current version of the O2 plugin (clone this repo to the plugins directory).
- View the site and click on the three dot menu at the top right of a post, followed by the Shortlink option. 
- Notice that this takes you to the post as you're visiting the short link.
- Now activate this version of the plugin by switching the branch to `add/clipboard-copy-on-shortlink`
- Click on the Shortlink option again, and the icon should animate as the short link is copied to the system clipboard.
- Have fun pasting your short link in a text editor, or wherever you like!
